db/migrate/20200115232918_add_user_id_to_kudos.rb

```diff
@@ -0,0 +1,34 @@
+class AddUserIdToKudos < ActiveRecord::Migration[5.1]
+  def up
+    if Rails.env.staging? || Rails.env.production?
+=begin
+      pt-online-schema-change -uroot --alter \
+      "ADD COLUMN user_id INT, ADD INDEX index_kudos_on_user_id (user_id)" \
+      D=$DATABASE,t=kudos --chunk-size=10k --max-flow-ctl 0 \
+      --pause-file /tmp/pauseme --max-load Threads_running=25 \
+      --critical-load Threads_running=400 --set-vars innodb_lock_wait_timeout=2 \
+      --alter-foreign-keys-method=auto --execute
+      insert into schema_migrations (version) values ("20200115232918")
+=end
+    else
+      add_column :kudos, :user_id, :int
+      add_index :kudos, :user_id
+    end
+  end
+
+  def down
+    if Rails.env.staging? || Rails.env.production?
+=begin
+      pt-online-schema-change -uroot --alter \
+      "DROP COLUMN user_id, DROP INDEX index_kudos_on_user_id (user_id)" \
```

Comment:
This is unlikely to ever come up, but: I don't think the syntax on this `DROP INDEX` statement is right. You shouldn't include the parenthesized column name the way that you did for `ADD INDEX`.

(Also, I'm not sure you need the call to `DROP INDEX`, because the only column that index includes is `user_id`, which is being dropped. So you might be able to get away with removing the `DROP INDEX` statement entirely.)